plottr/plot/base.py

```diff
@@ -264,6 +270,11 @@ class ComplexRepresentation(LabeledOptions):
     #: magnitude and phase
     magAndPhase = "Mag/Phase"
 
+    #: Natural Logarithmic magnitude and phase
+    log10_MagAndPhase = "20*log10(Mag)/Phase"
```

Comment:
I would just rename it to logMag/Phase or something like that. Like this seems to long for me